### ISSUE TITLE
demo apk does not launch

### ISSUE THREAD

**Issue by qrntz** [ocsigen/ocsigen-start#596]
![photo_2020-05-25_20-17-11](https://user-images.githubusercontent.com/802109/82838631-749eac80-9ed5-11ea-99a0-04eea1b7241d.jpg)

**Comment by balat** [comment]
Hi,
This is typically when you forgot `make chcp`
or when you did `make chcp` with wrong arguments (APP_SERVER etc).

**Comment by qrntz** [comment]
thanks for your reply @balat. i don't think it's in the docs though?

i have moved on by now but it might be useful to other people to document it

**Comment by ilankri** [comment]
Hello @qrntz,

Where does the faulty APK come from? Have you downloaded it from the link *APK for Android* on [this page](https://ocsigen.org/ocsigen-start/latest/manual/intro) or have you built it from [the source](https://github.com/ocsigen/osdemo)?

**Comment by qrntz** [comment]
yes, it was the APK from the website, i should have clarified that. since this issue was reported more than a year ago i don't know whether it still exists.

**Comment by ilankri** [comment]
Hello @qrntz,

The Android app available on [Google Play store](https://play.google.com/store/apps/details?id=com.osdemo.mobile) does work. We will fix the APK downloadable on [this page](https://ocsigen.org/ocsigen-start/latest/manual/intro).

Thanks for the report!
